<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteUser</class>
 <widget class="QDialog" name="DeleteUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>571</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delete User</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="idTextbox">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="idLabel">
      <property name="text">
       <string>Search ID:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>380</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>571</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="labelAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <item row="2" column="0">
     <widget class="QLabel" name="accountNameLabel">
      <property name="text">
       <string>Account Name: </string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="fNameLabel">
      <property name="text">
       <string>First Name:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="lNameLabel">
      <property name="text">
       <string>Last Name:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="foundFName">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="foundLName">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="foundAccountName">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="phoneLabel">
      <property name="text">
       <string>Phone Number:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="emailLabel">
      <property name="text">
       <string>Email Address:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="addressLabel">
      <property name="text">
       <string>Address:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="zipLabel">
      <property name="text">
       <string>Zip Code:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="cityLabel">
      <property name="text">
       <string>City: </string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="foundCity">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="foundZip">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="foundAddress">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="foundEmail">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="foundPhone">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="searchButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DeleteUser</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DeleteUser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>212</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
